### DISCUSSION TITLE
Add arm64 Apple Mac M1 architecture to the build matrix

### DISCUSSION THREAD

**Discussion by george-polevoy** [InteractiveComputerGraphics/SPlisHSPlasH#251] [Q&A]
I'd like to use SPlisHSPlasH on Mac M1. Looks like there is no package for Mac M1 arm64 architecture.

```
pip install pysplishsplash
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement pysplishsplash (from versions: none)
ERROR: No matching distribution found for pysplishsplash
```

**Comment by janbender** [comment]
There exists no Python wheel for Mac since we have no Mac system to build and test this. However, there is a branch which tries to fill this gap: 

https://github.com/ruberith/SPlisHSPlasH/tree/fix-macos
